.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
Should we have a virtualenv & README notes to support this tool? This doesn't have to be done right now.

Reply:
Medium-term, I think we'll probably want to set up `tox` to automate our tests, but I haven't done tox configuration before, so I figured I'd start simple.